VCFv4.4.draft.tex

```diff
@@ -387,11 +388,19 @@ \subsubsection{Fixed fields}
 
 \begin{itemize}
 \renewcommand{\labelitemii}{$\circ$}
-\item END: End reference position (1-based), indicating the variant spans positions POS--END on reference/contig CHROM.
-Normally this is the position of the last base in the REF allele, so it can be derived from POS and the length of REF, and no END INFO field is needed.
-However when symbolic alleles are used, e.g.\ in gVCF or structural variants, an explicit END INFO field provides variant span information that is otherwise unknown.
-
-This field is used to compute BCF's {\tt rlen} field (see~\ref{BcfSiteEncoding}) and is important when indexing VCF/BCF files to enable random access and querying by position.
+\item LEN / END: Fields used to explicitly specify the end position of the variant on the reference/contig CHROM.
+Normally this is the position of the last base in the REF allele, so it can be derived from POS and the length of REF, and no additional INFO field is needed.
+However when symbolic alleles are used, e.g.\ in gVCF or structural variants, an additional explicit field provides variant span information that is otherwise unknown.
+These fields are used to compute BCF's {\tt rlen} field (see~\ref{BcfSiteEncoding}) and are important when indexing VCF/BCF files to enable random access and querying by position.
+
+  \begin{itemize}
+   \item END: End reference position (1-based), indicating the variant spans positions POS--END on reference/contig CHROM.
+   END is deprecated and has been replaced with LEN.
+   \item LEN: The length of the variant's alignment to the reference indicating the variant spans positions POS--(LEN - 1) on reference/contig CHROM.
+  \end{itemize}
+
+At most one of LEN or END should be present on each variant line.
+If both are present they must be consistent with each other.
```

Comment:
- here's a constraint
- if you break the constraint, at least be nice

maybe the second part can be removed?

Reply:
The first is a suggestion not a constraint.  It could be changed to a constraint though.